components/automate-chef-io/content/docs/configuration.md

```diff
@@ -698,6 +710,37 @@ Then run `chef-automate config patch </path/to/your-file.toml>` to deploy your c
 # name = "automate"
 ```
 
+#### Configuring An External PostgreSQL Database
+To use Chef Automate with an external PostgreSQL cluster, create a `.toml` file containing the following partial configuration.
+Uncomment and change settings as needed, then run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
+
+Note that no data will be migrated from the built-in PostgreSQL service; therefore, this operation should only be performed on new Chef Automate deployments. If you would like to migrate an existing Chef Automate deployment to an external cluster, please contact support.
+
+```toml
+[global.v1.external.postgresql]
+enable = true
+nodes = ["<pghostname1>:<port1>", "<pghostname2>:<port2>", "..."]
+
+# To use postgres with SSL, uncomment and fill out the following:
+# [global.v1.external.postgresql.ssl]
+# enable = true
+# root_cert = """$(cat </path/to/root/cert.pem>)"""
+
+[global.v1.external.postgresql.auth]
+scheme = "password"
+
+[global.v1.external.postgresql.auth.password.superuser]
+username = "<admin username>"
+password = "<admin password>"
+[global.v1.external.postgresql.auth.password.dbuser]
+username = "<dbuser username>"
+password = "<dbuser password>"
+
+[global.v1.external.postgresql.backup]
+enable = true
+
+```
+
```

Comment:
I _think_ we are OK not restarting the world in these directions because the restart-workaround code will restart all pgsql clients.  However, I think in general we might still want to do a full service restart so that database initialization code in the various services run again.